README.md

```diff
@@ -129,7 +173,7 @@ The connector will go through the left_joins in order, looking them up and mergi
 
 #### Selecting Whole Models Instead of Fields
 
-Instead of specifying the precise fields you want, you can instead include the entire joined model in your model.
+Instead of specifying the precise fields you want, you can instead include the entire joined model in your model. The property "name" should *not* be used.
```

Comment:
"...model in your model as a single named field. "

Also "should" implies "can". Make it more of a positive, e.g. 
"To create a field that joins a whole model as a named field you create a field in your composite connector that has the type "Object" and the name of the source model only:"

Or something like that.

Also call out that this is really only for use as a 1:1 mapping.